### PR TITLE
Making shear and P wave speed user defined quantities for seismic for…

### PR DESCRIPTION
**Pull request by sveerara**
…ce and non reflecting BC #183

### CODE REVIEW

**Comment by cbolisetti** [comment]
@sveerara this looks good, but you also need to input the density of the underlying bedrock (apart from the S and P wave velocities) and use that density in the computation of the shear forces and the damper properties for the nonreflecting boundary.

File: src/actions/SeismicForceAction.C [hunk 1]
```diff
@@ -48,6 +48,10 @@ validParams<SeismicForceAction>()
                                              "input_component.");
   params.addParam<Real>(
       "alpha", 0.0, "The alpha parameter required for HHT time integration scheme.");
+  params.addRequiredRangeCheckedParam<Real>(
+      "p_wave_speed", "p_wave_speed>0.0", "P-wave speed of the material.");
```
Comment (cbolisetti) [inline src/actions/SeismicForceAction.C]: Can you please change this to 'P-wave speed of the underlying bedrock.'?

**Comment by sveerara** [comment]
@aeslaughter, can you please merge this PR?

**Comment by aeslaughter** [comment]
Sorry about not merging this sooner, I stopped paying attention because you assigned @cbolisetti. FYI, if your PR is approved by someone else you can merge it yourself.

**Comment by sveerara** [comment]
Thanks @aeslaughter. That's good to know.

**Pull request merged**